### PR TITLE
Team8 geocoder testing

### PR DESCRIPTION
**Pull request by rjwoo**
Added VCR testing to the organizations_spec.rb for geocoding along with adding the global.yml to the .gitignore.

### CODE REVIEW

**Comment by tkbeili** [comment]
@ryan please rebase for me to review

**Comment by rjwoo** [comment]
@tkbeili done, everything should be good to go let me know if you need anything else looked after

File: app/views/organizations/show.html.erb [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 
   <!-- Company "Avatar" -->
-  <% if @organization.image.present? %>
+  <%# if @organization.image.present? %>
     <%= image_tag @organization.image.url(:large) %>
-  <% end %>
+  <%# end %>
```
Comment (tkbeili) [inline app/views/organizations/show.html.erb]: Please remove commented out code

File: config/initializers/meetup_client.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 MeetupClient.configure do |config|
-  config.api_key = "52d143b1f4664496c35739a1a5467"
+  config.api_key = "266a521f17733d5f14104c469596176"
```
Comment (tkbeili) [inline config/initializers/meetup_client.rb]: Can you please make sure this uses an `ENV` variable?

File: spec/controllers/events_controller_spec.rb [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
 
 RSpec.describe EventsController, type: :controller do
+  describe "#index" do
+    it "renders the index template" do
+      get :index
+      expect(response).to render_template(:index)
+    end
 
+    it "crawls for new meet up events using task job" do
+      Event.destroy_all
```
Comment (tkbeili) [inline spec/controllers/events_controller_spec.rb]: Why you have a `destroy_all` as the first thing in this spec

Comment (rjwoo) [inline spec/controllers/events_controller_spec.rb]: We checked the previous commits, I believe Ethan was working on this /spec/controller

Comment (tkbeili) [inline spec/controllers/events_controller_spec.rb]: Ok then please try to find out the reasons as it doesn't seem necessary to me. Alternatively you can test without it.

**Pull request merged**